dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -1217,8 +1217,16 @@ def _run_test_command(
     perform_environment_checks()
     if skip_providers:
         ignored_path_list = [
-            f"--ignore=providers/tests/{provider_id.replace('.','/')}"
-            for provider_id in skip_providers.split(" ")
+            # TODO(potiuk): remove the old ways once we migrate all providers to the new structure
```

Comment:
NICE one !

Reply:
Actually a bit unfair to play the ball into your field but I thought copy/paste the comment is best for consistency for later cleanup. Can help there of course!

Reply:
That's OK. I want to remove that code ... It will add to my "removed code" stats.